crates/deterministic-wasi-ctx/Cargo.toml

```diff
@@ -11,12 +11,13 @@ categories = ["wasm"]
 [dependencies]
 anyhow = "1.0.56"
 async-trait = "0.1.53"
-cap-std = "0.25.2"
-wasi-common = ">= 0.34.0, < 1.1.0"
+cap-std = "0.26.1"
```

Comment:
It looks like this isn't even used anymore?

Reply:
Correct, removed from the dependencies.